src/metabase/models/permissions.clj

```diff
@@ -255,6 +255,15 @@
                                (and "schema/"
                                     (opt (and path-char "*/"
                                               (opt #"table/\d+/"))))))))
+               ;; any path starting with /general is a permission to access certain features of admin settings page
+               ;; /general/setting/    -> permissions to access /admin/settings page
+               ;; /general/monitoring/ -> permissions to access tools, audit and troubleshooting
+               ;; /general/subscription/ -> permisisons to create/edit subscriptions and alerts
+               (and "general/"
```

Comment:
Even though we could ignore the `/general/` prefix and it'll work just fine, but I'm still adding it to make the path clearer bc we might have more `general` permissions later on.